libs/execution/src/lib/blocks/block-executor-registry.ts

```diff
@@ -19,35 +20,35 @@ import {
   getOutputType,
 } from './composite-block-executor';
 
-export const blockExecutorRegistry = new Registry<BlockExecutorClass>();
-
-export function registerBlockExecutor(executorClass: BlockExecutorClass) {
-  blockExecutorRegistry.register(executorClass.type, executorClass);
-}
-
-export function getRegisteredBlockExecutors(): BlockExecutorClass[] {
-  return [...blockExecutorRegistry.getAll()];
+export function getBlockExecutorClass(
```

Comment:
Wouldn't this be more natural if the method was defined on the execution extension? E.g. you'd use 

```
const execExtension = new StdExecExtension(); 
execExtension.getBlockExecutorClass("blockname");
```

instead of having to create an extension object and then calling a global function with that object as param as you do later in the stdextension test.

Reply:
I wonder how that would play out when loading multiple extensions in the future... I think there is actually a need for a central component where you bring all the different extensions together, isn't it?

Or am I missing something?
Obviously, premature optimization is the source of all evil, so we can also remove it for now and introduce it later on (but in a separate PR?).

Reply:
Ah, I think I misunderstood your concern. Working on it!